releases/note-source/v18_1_0.rst

```diff
@@ -0,0 +1,52 @@
+.. _release-v18-1-0:
+
+RFC-618 Release (v18_1_0)
+=============================
+
++-------------------------------------------+------------+
+| Source                                    | Identifier |
++===========================================+============+
+| Git tag                                   | 18.1.0     |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v18\_1\_0  |
++-------------------------------------------+------------+
+
+- :ref:`release-v18-1-0-functionality`
+- :ref:`release-v18-1-0-interface`
+- :ref:`release-v18-1-0-pending-deprecations`
+- :ref:`release-v18-1-0-deprecations`
+
+*See also:*
+
+- :doc:`Installation instructions <../install/index>`
+- :doc:`Known issues </known-issues>`
+- `18.0.0 Characterization Metric Report (DMTR-141) <https://ls.st/DMTR-141>`_
+- `18.0.0 Doxygen Documentation <http://doxygen.lsst.codes/stack/doxygen/xlink_master_2019_06_08_08.07.58/>`__
+
+For a complete list of changes made, see :ref:`release-v18-1-0-tickets`.
+
+.. _release-v18-1-0-functionality:
+
+Major Functionality Changes
+---------------------------
+
+No functional changes implemented in this release
+
+Please note that eigen version jumped from 3.3.4.lsst1-1-g221959b to 3.3.7 due to a tag been added after 18.0.0 but forgotten when DM-18895 was implemented.
+
+Significant Interface Changes
+-----------------------------
```

Comment:
These headings seem quite redundant. Since this is a minor release, there *cannot* be any interface changes, and I'm pretty sure the same applies to deprecations.

While I understand that you are following the pattern established for major releases, I would suggest that for minor releases a simpler template is in order. Perhaps a single page, which just lists the reason(s) for providing the release and any other important notes (like the eigen changes).